projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3949,7 +3949,7 @@ public function admin_page_load() {
 					$url = $this->build_connect_url( true, $redirect, $from );
 
 					if ( ! empty( $_GET['onboarding'] ) ) {
-						$url = add_query_arg( 'onboarding', $_GET['onboarding'], $url ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+						$url = add_query_arg( 'onboarding', rawurlencode_deep( wp_unslash( $_GET['onboarding'] ) ), $url ); // WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Did I make a bad suggestion before?
```suggestion
						$url = add_query_arg( 'onboarding', rawurlencode_deep( wp_unslash( $_GET['onboarding'] ) ), $url ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```